### ISSUE TITLE
Document org-wide process for offtopic troll issue response

### ISSUE THREAD

**Issue by sftim** [kubernetes/community#5380]
### Organization or repository
[`kubernetes/org`](https://github.com/kubernetes/org)

### Describe the issue
I'd like the README at https://github.com/kubernetes/org#readme to signpost people (me!) to advice about how a maintainer can and should respond if offtopic troll issues come up.
(the response is likely to be to lock the issue, but it's a rare enough concern that we might not remember the details)

/kind feature
/sig contributor-experience
I think

**Comment by Nivedita-coder** [comment]
Hi @sftim , I would like to work on this issue, can you help me with what I need to add and where? Thanks

**Comment by celanthe** [comment]
Hi @Nivedita-coder--I'd be happy to guide you on how to move forward with this if you're still interested in taking it on! :) @sftim would that work for you?

**Comment by Nivedita-coder** [comment]
> Hi @Nivedita-coder--I'd be happy to guide you on how to move forward with this if you're still interested in taking it on! :) @sftim would that work for you?

Yes, I'm interested. It will be great for me ,Thanks @celanthe! :)

**Comment by celanthe** [comment]
An update on this, I met with @Nivedita-coder yesterday 1/27/21 via Zoom and walked her through how to structure a new PR that will address this issue and includes the changes suggested here! :)

**Comment by spiffxp** [comment]
/priority important-longterm
/milestone v1.23
/help
@celanthe @Nivedita-coder any progress on this? please /remove-help and /assign yourselves if you're still working on this
